src/asset-details/AssetDetailsCard.js

```diff
@@ -33,7 +33,7 @@ export default class AssetDetailsCard extends Component {
 				}
 				<br />
 				<h5><M m="Durations" /></h5>
-				<AssetDetailsDurations durations={durations} />
+				{durations && <AssetDetailsDurations durations={durations} />}
```

Comment:
Should the AssetDetailsDurations handle this?
In render if !durations return null